docs/architecture-docs.md

```diff
@@ -0,0 +1,29 @@
+# Ground Android app architecture
+
+The Ground Android app is written in Java and uses the [Android Jetpack](https://developer.android.com/jetpack) and [RxJava](https://github.com/ReactiveX/RxJava) libraries, among others. This page provides an overview of the high-level app architecture, including key components and libraries.
+
+<!-- Editable image source: https://docs.google.com/drawings/d/1UjetRJsudLHg3YsWaf0sKqHfmrPNNBH4UKI7Ivajx-0/ -->
+![Ground Android app architecture diagram](android-architecture-diagram.png)
+
+## Components
+
+The app is comprised of the following components:
+
+- **MainActivity**: Ground is a single-activity app. The main [activity](https://developer.android.com/guide/components/activities/intro-activities) contains all other app components.
+- **Fragments**: Each part of the app interface is defined as a [fragment](https://developer.android.com/guide/components/fragments). Examples of fragments are the home screen and the map interface.
```

Comment:
optional nit: `interface` -> `user interface` (since Java supports interfaces as a programming concept as well).

Reply:
Changed it to UI. Thanks